client/protocols/unlock.py

```diff
@@ -159,12 +187,15 @@ def interact(self, unique_id, case_id, question_prompt, answer, choices=None, ra
                 'prompt': question_prompt,
                 'answer': input_lines,
                 'correct': correct,
+                'treatment group id': tg_id,
+                'misU count': spen_dict,
```

Comment:
`misU` stands for misunderstanding.

On a different note @spnchian @haydensheung , please use an underscore instead of a space.